normits_demand/models/forecasting/ntem_forecast.py

```diff
@@ -642,6 +656,7 @@ def grow_all_matrices(
     matrices: NTEMImportMatrices,
     growth: TEMProTripEnds,
     output_folder: Path,
+    scenario: str = "TEM",
```

Comment:
Does this need to have a default value? I'm not convinced this makes obvious sense.
This is probably linked to the same question I have in `run_forecast`

Reply:
Yes should probably just be none for TEM forecasting.